### PR TITLE
[CLASSIFICATION] Fix multilabel case in classification model API wrapper

### PR DESCRIPTION
**Pull request by sovrasov**
This bug is not covered by the ote ci for now, so there is no ci job number.

### CODE REVIEW

**Comment by sovrasov** [comment]
run ote_sdk tests

**Comment by sovrasov** [comment]
BMM 27143

**Comment by sovrasov** [comment]
run ote_sdk tests

**Comment by sovrasov** [comment]
run ote_sdk tests

**Comment by sovrasov** [comment]
run ote_sdk tests

**Comment by sovrasov** [comment]
run ote_sdk tests

**Comment by sovrasov** [comment]
BMM 27417 - condicionally passed

**Comment by sovrasov** [comment]
run ote_sdk tests

File: ote_sdk/ote_sdk/utils/labels_utils.py [hunk 1]
```diff
@@ -24,26 +23,3 @@ def get_empty_label(label_schema: LabelSchemaEntity) -> Optional[LabelEntity]:
     if empty_candidates:
         return empty_candidates[0]
     return None
-
-
-def get_leaf_labels(label_schema: LabelSchemaEntity) -> List[LabelEntity]:
```
Comment (Ilya-Krylov) [inline ote_sdk/ote_sdk/utils/labels_utils.py:29]: why?

File: ote_sdk/ote_sdk/utils/labels_utils.py [hunk 2]
```diff
@@ -24,26 +23,3 @@ def get_empty_label(label_schema: LabelSchemaEntity) -> Optional[LabelEntity]:
...
-    leaf_labels = []
-    all_labels = label_schema.get_labels(False)
-    for lbl in all_labels:
-        if not label_schema.get_children(lbl):
-            leaf_labels.append(lbl)
-
-    return leaf_labels
-
-
-def get_ancestors_by_prediction(
```
Comment (Ilya-Krylov) [inline ote_sdk/ote_sdk/utils/labels_utils.py:42]: why?

File: ote_sdk/ote_sdk/utils/labels_utils.py (hunk 1 above)
Comment (sovrasov) [reply ote_sdk/ote_sdk/utils/labels_utils.py:29]: This functionality was replaced by label resolver

File: ote_sdk/ote_sdk/utils/labels_utils.py (hunk 2 above)
Comment (sovrasov) [reply ote_sdk/ote_sdk/utils/labels_utils.py:42]: This functionality was replaced by label resolver

**Comment by sovrasov** [comment]
run ote_sdk tests

**Pull request merged**